arduinolux/src/Debug/DebugTest.cpp

```diff
@@ -0,0 +1,14 @@
+#include"../StringConverter.h"
+#include <string>
+#include <iostream>
+
+int main() {
+	std::string testString = "Hello there world!";
+	do {
+		char* convertedString = StringConverter::ExtractCharArrayFromString(testString);
+
+		std::cout << convertedString << "\n";
```

Comment:
Cool. I'm not exactly clear on where you're intending to tie in to the existing translux code, so I'm not sure what you mean by "hard to validate it for compatibility with the translux"?

Are you wanting to use a different microcontroller instead of the nano, and re-use the existing code but for new "brain"? Are we talking about wanting to call rendermsgbits with std::string's instead of char[4][32]?

I would just change rendermsgbits' signature to take 4 const char *, and validate that strlen on all of them is <= 32, and fix rendermsgbits to just use up to the first \0 character in each line. Or am I missing something?